### ISSUE TITLE
TypeError when opening extension settings.

### ISSUE THREAD

**Issue by djohts** [xKippi/gnome-shell-extension-sane-airplane-mode#1]
```
TypeError: inputWidget.append is not a function

Stack trace:
  _init/addRow@/home/alex/.local/share/gnome-shell/extensions/sane-airplane-mode@kippi/prefs.js:72:33
  _init@/home/alex/.local/share/gnome-shell/extensions/sane-airplane-mode@kippi/prefs.js:87:9
  buildPrefsWidget@/home/alex/.local/share/gnome-shell/extensions/sane-airplane-mode@kippi/prefs.js:99:18
  _init@resource:///org/gnome/Shell/Extensions/js/extensionsService.js:207:40
  OpenExtensionPrefsAsync/<@resource:///org/gnome/Shell/Extensions/js/extensionsService.js:122:28
  asyncCallback@resource:///org/gnome/gjs/modules/core/overrides/Gio.js:132:13
  run@resource:///org/gnome/Shell/Extensions/js/dbusService.js:175:20
  main@resource:///org/gnome/Shell/Extensions/js/main.js:19:13
  run@resource:///org/gnome/gjs/modules/script/package.js:222:19
  start@resource:///org/gnome/gjs/modules/script/package.js:206:5
  @/usr/share/gnome-shell/org.gnome.Shell.Extensions:1:17
```
```
$ uname -a
Linux pcdjoh 5.15.0-41-generic #44~20.04.1-Ubuntu SMP Fri Jun 24 13:27:29 UTC 2022 x86_64 x86_64 x86_64 GNU/Linux
$ gnome-shell --version
GNOME Shell 3.36.9
```

**Comment by AntiKippi** [comment]
Thank you for reporting this. I see you're using GNOME Shell 3.36.9, how did you get to install the extension there in the first place  since it only (officially) supports GNOME 40+? I'll have a look at it anyway as soon as I get to it.

**Comment by djohts** [comment]
Hey, sorry that it took me this long to reply 😅. I installed it from [extensions.gnome.org](https://extensions.gnome.org/extension/4604/sane-airplane-mode/) using the Gnome browser extension. Just checked shell version list on the website, didn't notice it then lol.

**Comment by AntiKippi** [comment]
Fixed with [3db0abf](https://github.com/AntiKippi/gnome-shell-extension-sane-airplane-mode/commit/3db0abf6ec90dc18443aaab717643d59adc54a24)
